src/librustdoc/clean/utils.rs

```diff
@@ -589,9 +589,9 @@ pub(crate) fn attrs_have_doc_flag<'a>(
 /// so that the channel is consistent.
 ///
 /// Set by `bootstrap::Builder::doc_rust_lang_org_channel` in order to keep tests passing on beta/stable.
-pub(crate) const DOC_RUST_LANG_ORG_CHANNEL: &str = env!("DOC_RUST_LANG_ORG_CHANNEL");
+pub(crate) const DOC_RUST_LANG_ORG_VERSION: &str = env!("DOC_RUST_LANG_ORG_VERSION");
 pub(crate) static DOC_CHANNEL: Lazy<&'static str> =
```

Comment:
Can you rename this to `RUSTDOC_VERSION` too? For consistency with the above, but not `DOC_VERSION` because it could be confused with the current crate's version.